src/Components/Dashboard/Dashboard.js

```diff
@@ -6,10 +6,14 @@ import humidityIcon from '../../assets/humidity.png';
 import pressureIcon from '../../assets/pressure.png';
 import windIcon from '../../assets/wind.png';
 import emptyFavIcon from '../../assets/star-empty.png';
+import {convertToFahrenheit, convertWindToCardnialDirection, convertMsToMph} from '../../utilities.js';
 
 
-function Dashboard(props) {
-
+function Dashboard({location}) {
+const locationAqi = location.current.pollution.aqius;
+const tempInFahrenheit = convertToFahrenheit(location.current.weather.tp);
```

Comment:
Wait are these all built in JS native methods?! SICK! The more you know, huh?

Reply:
Just kidding, I'm seeing now that these methods live in utilities.js. Still SICK! Is that the best place for them to live? I don't really have any ideas on a better place for them to live at the moment but just wondering.